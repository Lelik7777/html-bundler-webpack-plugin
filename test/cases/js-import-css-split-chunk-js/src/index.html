<!DOCTYPE html>
<html>
<head>
  <link href="./css/style.css" rel="stylesheet">
  <script src="./js/app.js"></script>
</head>
<body>
  <h1>Home</h1>
  <div class="lib">.lib</div>
  <div class="app">.app</div>
  <div class="main-a">.main-a</div>
  <div class="main-b">.main-b</div>
</body>
</html>