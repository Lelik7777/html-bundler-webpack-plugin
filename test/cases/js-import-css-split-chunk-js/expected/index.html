<!DOCTYPE html>
<html>
<head>
  <link href="css/style.bundle.css" rel="stylesheet">
  <link href="css/app.bundle.css" rel="stylesheet">
<script src="js/runtime.bundle.js"></script>
<script src="js/107.bundle.js"></script>
<script src="js/671.bundle.js"></script>
<script src="js/192.bundle.js"></script>
<script src="js/450.bundle.js"></script>
<script src="js/app.bundle.js"></script>
</head>
<body>
  <h1>Home</h1>
  <div class="lib">.lib</div>
  <div class="app">.app</div>
  <div class="main-a">.main-a</div>
  <div class="main-b">.main-b</div>
</body>
</html>